packages/extension-browser/tests/end-to-end.ts

```diff
@@ -42,6 +58,43 @@ const findBackgroundScriptPage = async (browser: Browser): Promise<Page> => {
     return await bgTarget.page();
 };
 
+/**
+ * Find the Puppeteer `Page` associated with the devtools panel
+ * for the webhint browser extension.
+ *
+ * Needed because Puppeteer doesn't expose the devtools as a page.
+ *
+ * @param browser The Puppeteer `Browser` instance to search.
+ * @returns The found devtools panel for the extension.
+ */
+const findWebhintDevtoolsPanel = async (browser: Browser): Promise<Frame> => {
+    const targets = await browser.targets();
+    const devtoolsTarget = targets.filter((t) => {
+        return t.type() === 'other' && t.url().startsWith('chrome-devtools://');
+    })[0];
+
+    const devtoolsPage = await getPageFromTarget(devtoolsTarget);
+
+    await wait();
+
+    // Based on https://github.com/GoogleChrome/puppeteer/issues/3699#issuecomment-450526587
+    await devtoolsPage.keyboard.down('Control');
```

Comment:
Maybe add a comment for future generations explaining that this navigates to the left the selected panel on the devtools and that if we were to load multiple dev tools extensions it won't work? I know is more or less explained in the comment but I think it's better to have it in here just in case.